assembly/assembly-wsmaster-war/src/main/java/org/eclipse/che/api/deploy/WsMasterModule.java

```diff
@@ -183,6 +190,7 @@ protected void configure() {
     scmFileResolverResolverMultibinder.addBinding().to(GitlabScmFileResolver.class);
     scmFileResolverResolverMultibinder.addBinding().to(BitbucketServerScmFileResolver.class);
     scmFileResolverResolverMultibinder.addBinding().to(AzureDevOpsScmFileResolver.class);
+    scmFileResolverResolverMultibinder.addBinding().to(GitSshScmFileResolver.class);
```

Comment:
Should we add `RawDevfileUrlFactoryParameterResolver` here?

Reply:
`RawDevfileUrlFactoryParameterResolver ` does not implement `ScmFileResolver`